commons/AndroidManifest.xml

```diff
@@ -1,5 +1,5 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="fr.free.nrw.commons"
+    package="free.nrw.commons"
```

Comment:
I tried that initially but had a problem with it as it requires the addition of a 'fr' package in the middle of the package tiers and moving the 'free' package into it via refactor. The refactor seemed to break the links to the layout files, for example `resDialogText = R.string.crash_dialog_text` produces the error 'cannot resolve symbol R'.